px-k8s-sql-ha/sql.md

```diff
@@ -0,0 +1,53 @@
+In this step, we will deploy SQL Server. 
+
+### Step: Make sure Portworx is up
+Portworx doesn't run on the K8s master, so we'll check from one of the minions.
+
+```
+ssh node01 /opt/pwx/bin/pxctl status
+```{{execute T1}}
+
+### Step: Deploy SQL Server with Storage on-demand
+
+Let's create SQL Server and wait for it to be ready
+```
+kubectl create -f sqlserver-all.yaml
+./wait4-mssql.sh
+kubectl get pods
+```{{execute T1}}
+
+### Step:  Observe the Portworx Volume
+
+```
+ssh node01 /opt/pwx/bin/pxctl volume list
```

Comment:
Instead of volume list, just use `kubectl get pvc`. It will show the PVC as bound when volume is created.